src/views.py

```diff
@@ -73,79 +71,59 @@ def get_all(ticker):
         return new_data
 
 
-@app.route('/<ticker>/historical/max')
-def get_historical_all(ticker):
-    '''Retrieve all known historical data in 1 day increments for stock index'''
+@app.route('/<ticker>/historical/5_days')
```

Comment:
Changes here just involve rearranging the order of the routes in the code for easier readability, as well as removing code that was moved into retrieve_historical()